ffxivbot/handlers/QQCommand_bot.py

```diff
@@ -72,7 +72,7 @@ def QQCommand_bot(*args, **kwargs):
                     "所在窝：{}\n".format(WEB_BASE_URL.rstrip('/'))+\
                     "领养者：{}\n".format(bot.owner_id)+\
                     "群数量：{}\n".format(len(json.loads(bot.group_list)))+\
-                    "好友数量：{}\n".format(len(friend_list.get("friends", [])))+\

```

Comment:
Can you make the code compatible with the current database schema?

Reply:
The friend_list is array now because new api response is a JSON array.